src/Stryker.Core/Stryker.Core/Reporters/Html/Realtime/RealtimeMutantHandler.cs

```diff
@@ -25,6 +31,28 @@ public void CloseSseEndpoint()
     public void SendMutantTestedEvent(IReadOnlyMutant testedMutant)
     {
         var jsonMutant = new JsonMutant(testedMutant);
+
+        if (_server.HasClientsConnected)
```

Comment:
Could you use full if statements including braces

Reply:
Fixed